apply/driver/local.go

```diff
@@ -159,7 +159,7 @@ func (applier *Applier) reconcileCluster() error {
 	}
 	defer func() {
 		if err := applier.unMountClusterImage(); err != nil {
-			logrus.Warnf("failed to umount image(%s): %v", applier.ClusterDesired.ClusterName, err)
+			logrus.Warnf("failed to umount cluster(%s): %v", applier.ClusterDesired.Name, err)
```

Comment:
@kakaZhou719 @allencloud Help to consider if the ClusterName is needed. In new k8s/api version, the ClusterName is removed. IMO, we have the Cluster struct already, no need to have the ClusterName filed.

Reply:
agree with you. `cluster.Name` is meaningful to us rather than `cluster.ClusterName`

Reply:
+1

Reply:
I checked this. This is a weird logic.
The ClusterName is just used in this place.
I guess someone misunderstand the ClusterName and Name.